ArmaforcesMissionBot/Modules/Signups.cs

```diff
@@ -657,6 +657,40 @@ public async Task EditMission(int missionNo)
                 }
             }
         }
```

Comment:
I'd remove the index based edition as it's totally useless when we can just mention the channel.